test/functional/rpc_signrawtransaction.py

```diff
@@ -151,21 +154,24 @@ def script_verification_error_test(self):
     def witness_script_test(self):
         self.log.info("Test signing transaction to P2SH-P2WSH addresses without wallet")
         # Create a new P2SH-P2WSH 1-of-1 multisig address:
-        embedded_address = self.nodes[1].getaddressinfo(self.nodes[1].getnewaddress())
-        embedded_privkey = self.nodes[1].dumpprivkey(embedded_address["address"])
-        p2sh_p2wsh_address = self.nodes[1].addmultisigaddress(1, [embedded_address["pubkey"]], "", "p2sh-segwit")
+        eckey = ECKey()
+        eckey.generate()
+        embedded_privkey = bytes_to_wif(eckey.get_bytes())
+        embedded_pubkey = eckey.get_pubkey().get_bytes().hex()
+        p2sh_p2wsh_address = self.nodes[1].createmultisig(1, [embedded_pubkey], "p2sh-segwit")
         # send transaction to P2SH-P2WSH 1-of-1 multisig address
         self.nodes[0].generate(101)
         self.nodes[0].sendtoaddress(p2sh_p2wsh_address["address"], 49.999)
         self.nodes[0].generate(1)
         self.sync_all()
-        # Find the UTXO for the transaction node[1] should have received, check witnessScript matches
-        unspent_output = self.nodes[1].listunspent(0, 999999, [p2sh_p2wsh_address["address"]])[0]
```

Comment:
In commit "Avoid dumpprivkey and watchonly behavior in rpc_signrawtransaction.py" (f4f81c98dca6da030b84b5b4887cf3ef4b72baec)

Can commit description be updated to mention this change and reason listunspent is not longer used or checked? Change seems reasonable but I don't get it currently

Reply:
Updated the commit message. `listunspent` is not being used and is not the test target for this test. It is tested elsewhere. There is no need to attempt to bring this back for descriptor wallets.